src/common/DataConversionFlowGraph.cpp

```diff
@@ -128,7 +128,7 @@ Result DataConversionFlowGraph::configure(AudioStream *sourceStream, AudioStream
                                                                   actualSourceFramesPerCallback);
                 break;
             default:
-                LOGE("%s() Unsupported source caller format = %d", __func__, sourceFormat);
+                LOGE("%s() Unsupported source caller format = %s", __func__, oboe::convertToText(sourceFormat));
```

Comment:
In this case, we want to know what strange integer value was passed. So the convertToText() will probably not handle it. So better to cast to (int) and print with %d.